src/OpenTelemetry.Api/README.md

```diff
@@ -70,7 +71,40 @@ with the intent to produce continuous summaries of those measurements.
 
 _**Warning:** OpenTelemetry .NET has a prototype Metrics API implementation only
 and is not recommended for any production use. The API is expected to change
-heavily. Please check the [Metric support plan](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1501)._
+heavily. Please check the [Metric support
+plan](https://github.com/open-telemetry/opentelemetry-dotnet/issues/1501)._
+
+### Baggage API
+
+[Baggage
+API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md)
+allows users to add context to metric, traces, and logs. Baggage can be
+propagated out of proc using
+[Propagators](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md).
+OpenTelemetry SDK ships a BaggagePropagator and enables it by default.
```

Comment:
We need to mention that propagation is done using W3CBaggage standard, using the header prescribed from W3CBaggage spec.